lib/MusicBrainz/Server/Data/Editor.pm

```diff
@@ -303,7 +303,7 @@ sub update_email
         }
         else {
             $self->sql->do('UPDATE editor SET email=NULL, email_confirm_date=NULL
-                      WHERE id=?', $editor->id);
+                    WHERE id=?', $editor->id);
```

Comment:
intentional?  (doesn't appear to align things any better, not that it was good before.)

Reply:
Nope, reverting.